.gitattributes

```diff
@@ -1,2 +1,14 @@
 *.png filter=lfs diff=lfs merge=lfs -text
 *.jpg filter=lfs diff=lfs merge=lfs -text
+# vendor is marked as linguist-vendored by default, but this
+# only excludes the code from stats.
+#
+# Forcing it to be linguist-generated will also exclude it from diffs.
+#
+# See https://github.com/github-linguist/linguist/blob/master/docs/overrides.md#summary
+# for more info.
+api/ linguist-generated
+common/persistence/mock/ linguist-generated
```

Comment:
Why do we need to mention this explicitly; wouldn't that be covered by the last two mock-specific rules?

Reply:
Looks like they would be caught, I'll just RM that